Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1533,6 +1533,20 @@ The mini form task is used to test survey forms functionality (`ORKFormStep`).
     }
     
     {
+        /*
+         A text question with single-line text entry, text validation, and a URL keyboard.
+         */
+        ORKTextAnswerFormat *format = [ORKAnswerFormat textAnswerFormatWithValidationExpression:@"^(https?:\\/\\/)?([\\da-z\\.-]+)\\.([a-z\\.]{2,6})([\\/\\w \\.-]*)*\\/?$" validInputDescription:@"Enter a valid URL."];
```

Comment:
The regex needs to be a bit more defined.
Currently, it accepts "www.goo" as valid url.